crates/orderbook/src/dto/mod.rs

```diff
@@ -5,11 +5,18 @@ pub use {
     auction::{Auction, AuctionId, AuctionWithId},
     order::Order,
 };
-use {serde::Serialize, serde_with::serde_as};
+use {
+    number::serialization::HexOrDecimalU256,
+    primitive_types::U256,
+    serde::Serialize,
+    serde_with::serde_as,
+};
 
 #[serde_as]
 #[derive(Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct TokenMetadata {
     pub first_trade_block: Option<u32>,
+    #[serde_as(as = "Option<HexOrDecimalU256>")]
+    pub stored_native_price: Option<U256>,
```

Comment:
I'd say where the native price came from (current value or last one from the DB) doesn't matter for the consumer of the API.
```suggestion
    pub native_price: Option<U256>,
```